code/modules/resleeving/infocore.dm

```diff
@@ -67,7 +68,7 @@ var/datum/transhuman/infocore/transcore = new/datum/transhuman/infocore
 			MR.nif_software = null
 
```

Comment:
`if (!one_time) MR.one_time = FALSE`  perhaps? To handle if someone gets a one time backup, THEN gets an implant later?